ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKNavigationDelegate.swift

```diff
@@ -97,6 +97,19 @@ extension BrowserViewController: WKNavigationDelegate {
         hideReaderModeBar(animated: false)
       }
     }
+
+    if let tab = tab(for: webView) {
+      tab.redirectURLs = []
+    }
+    appendToRedirectChain(webView)
+  }
+
+  fileprivate func appendToRedirectChain(_ webView: WKWebView) {
+    // Append current WebView URL to redirect chain. The redirect chain MUST be
```

Comment:
`Append current WebView URL to redirect chain.` is not needed because the function name is well named.

Reply:
Fixed